finances.html

```diff
@@ -85,7 +85,7 @@ <h3 class="main-title">Finances</h3>
             </div>
 
             <div class="form-popup">
-                <form class="form-container" id="transaction-form" onsubmit="newTransaction(event)">
+                <form class="form-container" id="transaction-form" onsubmit="addOrUpdate(event)">
```

Comment:
I wanted to get away from changing the submit button behaviour so just have both actions go through the same function